cmake/options.cmake

```diff
@@ -2,9 +2,10 @@ set(CMAKE_EXPORT_COMPILE_COMMANDS "ON")
 set(CMAKE_MACOSX_RPATH 1)
 
 option(CMAKE_BUILD_TYPE "Choose the type of build, options are: Debug Release RelWithDebInfo MinSizeRel." "Release")
-option(VINECOPULIB_SHARED_LIB    "Pre-compiled version"              "OFF")
-option(WARNINGS_AS_ERRORS        "Compiler warnings as errors"       "OFF")
-option(OPT_ASAN                  "Use adress sanitizer (debug)"       "ON")
-option(BUILD_TESTING             "Build tests."                       "ON")
-option(CODE_COVERAGE             "Code coverage."                    "OFF")
-option(STRICT_COMPILER           "Stricter compiler warnings"        "OFF")
\ No newline at end of file
+option(VINECOPULIB_SHARED_LIB    "Pre-compiled version"              OFF)
```

Comment:
Ah, so CMake actually has boleans, I thought one had to use strings instead. Good to know.